libs/ui/input/helm/src/lib/hlm-input.directive.ts

```diff
@@ -45,7 +45,7 @@ type InputVariants = VariantProps<typeof inputVariants>;
 export class HlmInputDirective implements BrnFormFieldControl, DoCheck {
 	public readonly size = input<InputVariants['size']>('default');
 
-	public readonly error = signal<InputVariants['error']>('auto');
+	public readonly error = input<InputVariants['error']>('auto');
```

Comment:
@goetzrobin should be fixed now. Somehow I forgot to change error to `input` which caused the type error in storybook.